src/Sylius/Bundle/ThemeBundle/composer.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "sylius/theme-bundle",
+    "type": "symfony-bundle",
+    "description": "Themes management for Symfony2 projects.",
+    "keywords": ["themes", "theming"],
+    "homepage": "http://sylius.org",
+    "license": "MIT",
+    "authors": [
+        {
+            "name":     "Kamil Kokot",
+            "homepage": "http://kamil.kokot.me"
+        },
+        {
+            "name":     "Sylius project",
+            "homepage": "http://sylius.org"
+        },
+        {
+            "name":     "Community contributions",
+            "homepage": "http://github.com/Sylius/Sylius/contributors"
+        }
+    ],
+    "require": {
+        "php":                      ">=5.4",
+
+        "sylius/resource-bundle":   "0.14.*@dev",
```

Comment:
I can be wrong but the resource-bundle doesn't seem to be used by any of the commits.
Is it possible to use the ThemeBundle without having to install the ResourceBundle?